services/movie_session.py

```diff
@@ -0,0 +1,50 @@
+import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    query_set = MovieSession.objects.all()
+
+    if session_date:
+        session_date = datetime.datetime.strptime(
+            session_date, "%Y-%m-%d"
+        ).date()
+        return MovieSession.objects.filter(show_time__date=session_date)
+
+    return query_set
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def create_movie_session(movie_show_time: str,
+                         movie_id: int,
+                         cinema_hall_id: int) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
```

Comment:
The `show_time` parameter should be converted to a `datetime` object before being used in `MovieSession.objects.create`. This ensures that the `show_time` field is stored correctly in the database.